seed/views/column_list_settings.py

```diff
@@ -46,3 +55,92 @@ class ColumnListingViewSet(OrgValidateMixin, SEEDOrgCreateUpdateModelViewSet):
     pagination_class = None
     # force_parent = True  # Ideally the column list settings would inherit from the parent,
     # but not yet.
+
```

Comment:
General comments about these two methods and the customizations happening within them. 

My initial thought was to use a serializer, but before I even looked at if that was possible, I changed my mind as this injection of a ColumnListSetting shouldn't be tucked away into the serializer. In other words, it should be explicitly pointed out in these methods like you have them already.

Reply:
After discussing, it was both easier and preferably more obvious to implement these injections explicitly.